ONSdigital/design-system
# Remove simple-slack-notify and move to using github slack app

### What is the context of this PR?

`simple-slack-notify` has been archived, is no longer maintained and is now running on an old version of node. This PR makes the change of removing that and then I will set up the `pat-lib-notifications` channel to monitor these workflows using the github slack app.
As part of this I have made the decision to split the tests out into separate workflows due to a problem in slack with the naming of the workflow containing a comma. These were separate jobs anyway that didn't rely on each other so makes sense that they are separate anyway.

Part of the fix for this https://github.com/ONSdigital/design-team/issues/9#

**Once merged update branch protection and reconfigure slack integration**

### How to review this PR

- Set up a chat between you and the github app in slack
- Connect your github account by sending the message `/github signin` if not prompted
- Send`/github subscribe ONSdigital/design-system` to start monitoring the repo
- Send `/github subscribe ONSdigital/design-system workflows:{name:"Lighthouse CI accessibility checks", "Publish Design System to npm", "Run visual regression tests", "Run macro and script tests"}` to subscribe to the correct workflows.
- Compare the notifications you get with the ones in the `pat-lib-notifications` channel and see that all notifications are replicated.
- Unsubscribe again with the message `/github unsubscribe ONSdigital/design-system`

### Checklist

This needs to be completed by the person raising the PR.



- [x] I have selected the correct Project for this PR (Design System) and selected the correct status
